index.html

```diff
@@ -57,13 +57,18 @@ <h1 class="display-3">Bikes for Refugees</h1>
 						<p class="lead">Providing donated bikes and accessories to refugees and asylum seekers in Scotland.</p>
 						<hr class="my-4">
 						<div class="buttons">
-							<a class="btn btn-primary btn-lrg" href="#">Donate a bike today</a>
+							<a class="btn  btn-orange" href="#">Donate a bike today</a>
 							<a class="btn btn-secondary btn-lrg" href="#">Volunteer</a>
 						</div>
 					</div>
+					<section class="flex-container row">
+						  <div class="flex-elem col-md-3">check availability</div>
+						  <div class="flex-elem col-md-3">donate bikes</div>
+						  <div class="flex-elem col-md-3">volunteer with us</div>
+					</section>
 					<h2 class="heading-underline">Learn more</h2>
-					<div class="articles">
-						<article class="article">
+					<div class="articles row ">
+						<article class="article col-md-6">
 							<div class="article-title">
```

Comment:
A tip here: your `col` element should wrap around the content, but not have the content directly inside as the borders will touch. If you add a row`div` around your `article`, you will see your boxes don't touch anymore.